eslint-rules/lib/rules/no-direct-import.js

```diff
@@ -85,8 +87,45 @@ module.exports = {
       }
     }
 
+    function reportDirectRequire(node, rule) {
+      try {
+        const {applyAutofix, destination} = rule;
+        const message = getErrorMessage(rule, 'require');
+        const moduleImported = (node.init.object ? node.init.object.arguments : node.init.arguments)[0];
+        context.report({
```

Comment:
I don't like that we have two different `context.report` but I see we already have a precedent so... 🤷‍♂️

Reply:
I could change the implementation a little so there would be just one report. With something like this:
```javascript
function reportError(node, rule, textToReplace) {
  try {
    const {applyAutofix, destination} = rule;
    const message = getErrorMessage(rule, 'import');
    context.report({
      node,
      message,
      fix(fixer) {
        if (node && applyAutofix && destination) {
          return fixer.replaceText(textToReplace, `'${destination}'`);
        }
      }
    });
  } catch (err) {
    handleError(RULE_ID, err, context.getFilename());
}
```

Reply:
I combined the implementation

Reply:
Excelent!